addons/project-title-generator/addon.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Project title generator",
+  "description": "Adds a button to generate random titles for your projects.",
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://scratch.mit.edu/users/lisa_wolfgang/"
```

Comment:
Username capitalization is correct. I checked.

Reply:
Um they would know since its their username